Documentation/_exts/cilium_external_links.py

```diff
@@ -0,0 +1,20 @@
+from sphinx.writers.html import HTMLTranslator
+from sphinx.writers.html5 import HTML5Translator
+from sphinx.util.docutils import is_html5_writer_available
+
+# Makes all external links open in new tabs
+class PatchedHTMLTranslator(
+    HTML5Translator if is_html5_writer_available() else HTMLTranslator
+): 
+    def starttag(self, node, tagname, *args, **attrs):
+        if (
+            tagname == "a"
+            and "target" not in attrs
+            and (
+                "external" in attrs.get("class", "")
+                or "external" in attrs.get("classes", [])
+            )
+        ):
+            attrs["target"] = "_blank"
+            attrs["ref"] = "noopener noreferrer"
+        return super().starttag(node, tagname, *args, **attrs)
```

Comment:
Note the symbol that GitHub added at the end of the file: This is because the file doesn't have a newline at the end. This is likely due to how your editor is configured. Would be better to add one for consistency, please.

Reply:
Yes. I see it. Thanks for pointing this out.

Reply:
This issue is still present. I see you added a line, but it contains spaces (from auto-indent?). [Files should end with a newline symbol](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline). You probably want to check the settings of your editor. Vim should add the line by default (`eol` option), VSCode [has a relevant option](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files), if using something else there's a high chance you can also find a similar config option.

Reply:
So sorry for not checking after making the change. 

Just solved the issue. Thank you very much for the helpful links.

![Screenshot 2022-08-22 at 11 23 23](https://user-images.githubusercontent.com/42554056/185899578-d7743980-b337-44f5-b7ab-2bac3539f215.png)